components/layouts/mainLayout.tsx

```diff
@@ -32,19 +32,24 @@ export const MainLayout: FC<PropsWithChildren<MainLayoutProps>> = ({
   const navigation =
     userType === 'DOCTOR' ? doctorNavigation : patientNavigation
 
-  const userName = 'Tom Cook'
+  const { data: userLoggedIn } = useQuery(
+    ['getUser'],
+    async () => await getUser(),
+  )
+
+  const userName = userLoggedIn?.name
 
   const { mutate } = useMutation({
     mutationFn: logoutUser,
     onSuccess: () => {
       localStorage.removeItem('token')
       router.push('/login')
     },
-    onError: (error) => {
-      console.log('error' ,error)
-    }
+    onError: error => {
+      console.log('error', error)
```

Comment:
would be nice to have some kind of message/warning/state to show there's an error